community/bolt/src/test/java/org/neo4j/bolt/v1/runtime/integration/SessionAuthIT.java

```diff
@@ -51,7 +51,7 @@ public void shouldGiveCredentialsExpiredStatusOnExpiredCredentials() throws Thro
         session.init( "TestClient/1.0.0", map(
                 "scheme", "basic",
                 "principal", "neo4j",
-                "credentials", "neo4j" ), null, recorder );
+                "credentials", "neo4j" ), -1, null, recorder );
```

Comment:
Maybe use `TransactionIdStore.BASE_TX_ID` instead of all these `-1` because later is not really a valid tx id.